simphony/core/cuba.yml

```diff
@@ -480,4 +480,46 @@
   name: Material
   number: 68
   shape: [1]
-  type: uuid
\ No newline at end of file
+  type: uuid
+- description: Cutoff Distance
+  domain: []
+  key: CUTOFF_DISTANCE
+  name: CutoffDistance
+  number: 69
+  shape: [1]
+  type: double
+- description: Energy Well Depth
+  domain: []
+  key: ENERGY_WELL_DEPTH
+  name: EnergyWellDepth
+  number: 70
+  shape: [1]
+  type: double
+- description: Van Der Waals Radius
+  domain: []
+  key: VAN_DER_WAALS_RADIUS
+  name: VanDerWaalsRadius
+  number: 71
+  shape: [1]
+  type: double
+- description: Dielectric Contance
+  domain: []
+  key: DIELECTRIC_CONTANCE
+  name: DielectricContance
+  number: 72
+  shape: [1]
+  type: double
+- description: Lennard Jones
```

Comment:
I think the two material relations (`LennardJones` and `Coulomb`) could be moved out of the CUBA list.  

Maybe it would be best to just use what is defined in `material_relations.yml` to create a seperate enum.  So maybe add 'number' to 'material_relations.yml' and generate a seperate enum (called CUBA_MATERIAL_RELATIONS or MATERIAL_RELATIONS?) 

Then the MaterialRelation attribute `kind` would be switched from `kind: CUBA` to something like `kind:CUBA_MATERIAL_RELATIONS.`